RowField.h

```diff
@@ -51,6 +51,6 @@ typedef enum ReservedFields_ {
 
 /* Follow ReservedField entries with dynamic fields defined at runtime */
 #define ROW_DYNAMIC_FIELDS LAST_RESERVED_FIELD
-typedef int32_t RowField;
+typedef int32_t FieldID;
 
```

Comment:
If we were going to do this, the filename here would have to change.

But (as I said before you wrote this patch) - IMO "Field" is too generic a term.  Conceptually, there are individual 'fields' within Meters for example.

So I don't think we should go this route.

Reply:
Yes. And I plan the next commit would be to rename `RowField` to `Field` for the entire module.
Speaking of meters, the current code base refer to what you called 'fields' as 'items', so there is no confusion.
If you would complain that name being too generic, I would argue the same for the `Table` and `Row` class names.

Reply:
There is confusion here, for the reasons I've pointed out.

If we ask anyone what part of the htop UI relates to 'tables' and 'rows', they'll point out the exact right thing every time - trying to compare that to 'Fields' just doesn't work, sorry.

Its a NAK from me on this proposed naming change.

Reply:
@natoscott I think the 'Field' would refer to the table cell. And it's not weird to call it a 'Field' anyway.

https://en.wikipedia.org/wiki/Column_(database)

Reply:
Sure, I understand where you're coming from - but it is quite ambiguous even within htop.  Definitely not "wierd" though - 'too generic' was the phrase used.